Analysis/include/AnaTuple.h

```diff
@@ -180,12 +181,19 @@ class AnaTupleWriter {
         const FatJetCandidate* fat_jet_cand;
     };
 
+    struct BTagWeights {
+        std::map<DiscriminatorWP, std::map<UncertaintyScale, float>> weights;
+        std::map<DiscriminatorWP, std::map<UncertaintyScale, float>> weights_iter;
+        std::map<DiscriminatorWP, std::map<UncertaintyScale, float>> weights_iter_jes;
```

Comment:
- for iterative fit SF doesn't depend on WP, but there are multiple unc sources => `std::map<DiscriminatorWP, std::map<UncertaintyScale, float>>` -> `std::map<std::pair<UncertaintySource, UncertaintyScale>, float>`
- The direct effect of total JES on btag is relevant only for Total JES unc => `std::map<DiscriminatorWP, std::map<UncertaintyScale, float>>` -> `std::map<UncertaintyScale, float>`.